Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -189,7 +190,42 @@ function MiG_29A:defineCabinTempSwitch(identifier, device_id, arg_number, catego
 	end)
 end
 
+--- Adds a new push button control that sends a negative input
+--- @param identifier string the unique identifier for the control
+--- @param device_id integer the dcs device id
+--- @param command integer the dcs command
+--- @param arg_number integer the dcs argument number
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+--- @return Control control the control which was added to the module
+function MiG_29A:defineInvertedPushButton(identifier, device_id, command, arg_number, category, description)
+	local control = self:defineTumb(identifier, device_id, command, arg_number, 1, { -1, 0 }, nil, false, category, description)
+	control.api_variant = ApiVariant.momentary_last_position
+
+	return control
+end
+
 -- Stick
+local STICK = "Stick Controls"
+
+MiG_29A:definePushButton("STICK_TRIM_FWD", devices.HOTAS, 3004, 51, STICK, "Trim Switch FWD")
+MiG_29A:defineInvertedPushButton("STICK_TRIM_AFT", devices.HOTAS, 3005, 51, STICK, "Trim Switch AFT")
```

Comment:
you might check out something like defineRocker or similar for handling a multi-command control like this